packages/ra-ui-materialui/src/button/PrevNextButton.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Basic } from './PrevNextButton.stories';
+describe('<PrevNextButton />', () => {
+    beforeEach(() => {
+        window.scrollTo = jest.fn();
+    });
+    it('should render next prev button at position 4 with a total of 905', async () => {
```

Comment:
You should describe the feature you test regardless of the test data. Something like: "It should render the position of the current record in the list"